src/gui/application.cpp

```diff
@@ -777,8 +777,8 @@ void Application::openVirtualFile(const QString &filename)
     folder->implicitlyHydrateFile(relativePath);
     QString normalName = filename.left(filename.size() - virtualFileExt.size());
     auto con = QSharedPointer<QMetaObject::Connection>::create();
-    *con = QObject::connect(folder, &Folder::syncFinished, [con, normalName] {
-        QObject::disconnect(*con);
+    *con = connect(folder, &Folder::syncFinished, folder, [folder, con, normalName] {
```

Comment:
is the folder as target object really needed?

capturing it should be enough, no?

Reply:
yes but if I capture it and don't use it the compiler will complain about an unused capture...